package/gluon-core/src/respondd.c

```diff
@@ -0,0 +1,57 @@
+/*
+  Copyright (c) 2016, Leonardo Mörlein <git@irrelefant.net>
+  All rights reserved.
+
+  Redistribution and use in source and binary forms, with or without
+  modification, are permitted provided that the following conditions are met:
+
+    1. Redistributions of source code must retain the above copyright notice,
+       this list of conditions and the following disclaimer.
+    2. Redistributions in binary form must reproduce the above copyright notice,
+       this list of conditions and the following disclaimer in the documentation
+       and/or other materials provided with the distribution.
+
+  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+  DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+  FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+  DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+  OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+
+#include <respondd.h>
+
+#include <json-c/json.h>
+#include <libgluonutil.h>
+
+#include <string.h>
+
+
+static struct json_object * respondd_provider_nodeinfo(void) {
+	struct json_object *site = gluonutil_load_site_config();
+	if (!site)
+		return NULL;
+
+	struct json_object *domain_name = NULL;
+	json_object_object_get_ex(site, "domain_name", &domain_name);
+	if (domain_name)
+		json_object_get(domain_name);
+
+	json_object_put(site);
+
+	struct json_object *ret = json_object_new_object();
+	json_object_object_add(ret, "domain_name", domain_name);
```

Comment:
Is there a reason, why you start use the `gluon-core` for a `respondd.c`.
Should not that be in the package `gluon-respondd` ??

Reply:
FYI: As discussed earlier on IRC `domain_name` will be dropped and `system.domain_code` will be added.

Reply:
>Is there a reason, why you start use the gluon-core for a respondd.c.
> Should not that be in the package gluon-respondd ??
Done